apps/builder/app/builder/features/style-panel/sections/transforms/transform-utils.test.ts

```diff
@@ -0,0 +1,255 @@
+import { afterEach, describe, test, expect } from "@jest/globals";
+import {
+  addDefaultsForTransormSection,
+  handleDeleteTransformProperty,
+  handleHideTransformProperty,
+  isTransformPanelPropertyExists,
+  updateRotateOrSkewPropertyValue,
+  updateTransformTuplePropertyValue,
+} from "./transform-utils";
+import type { StyleInfo } from "../../shared/style-info";
+import {
+  FunctionValue,
+  toValue,
+  TupleValue,
+  type StyleProperty,
+  type StyleValue,
+} from "@webstudio-is/css-engine";
+import {
+  extractRotatePropertiesFromTransform,
+  parseCssValue,
+} from "@webstudio-is/css-data";
+
+let currentStyle: StyleInfo = {};
+const setProperty = (property: StyleProperty) => (value: StyleValue) => {
+  currentStyle[property] = { value };
+};
+const deleteProperty = (property: StyleProperty) => {
+  delete currentStyle[property];
+};
```

Comment:
I think it would be nicer to just wrap this into a function instead of resetting after each ... these resets are  often hard to understand

Reply:
const createSomething = () => {
  let currentStyle: StyleInfo = {};
const setProperty = (property: StyleProperty) => (value: StyleValue) => {
  currentStyle[property] = { value };
};
const deleteProperty = (property: StyleProperty) => {
  delete currentStyle[property];
};

return {setProperty, deleteProperty}
}

then you don't need beforeEach and reset